pkg/auth/providers/providers.go

```diff
@@ -192,6 +192,7 @@ func SearchPrincipals(name, principalType string, myToken v3.Token) ([]v3.Princi
 				return principals, err
 			}
 			principals = append(principals, localPrincipals...)
+			_ = localPrincipals
```

Comment:
What is this change for? I don't see it in the output in the issue

Reply:
Any of these are `ineffassign` reports, I've had a few passes so apparently not everything is in the first report.